src/replica/duplication/replica_duplicator.cpp

```diff
@@ -161,7 +161,7 @@ void replica_duplicator::start_dup_log()
     _load = std::make_unique<load_mutation>(this, _replica, _load_private.get());
 
     from(*_load).link(*_ship).link(*_load);
-    fork(*_load_private, LPC_REPLICATION_LONG_LOW, 0).link(*_ship);
+    fork(*_load_private, LPC_REPLICATION_LONG_COMMON, 0).link(*_ship);
```

Comment:
The reason of priority set to LPC_REPLICATION_LONG_LOW is to reduce the impact of duplication on normal read/write operations on the master cluster. There are some usage scenarios of the duplication, some user cares about the latency, some cares about the master cluster's thoughput.

How about add a configuration for it?

Reply:
I also want to add this configuration, but I haven't decided whether to make it a table-level configuration or a cluster-level configuration. 
At present, the duplicate_log_batch_bytes parameter is cluster-level, so should I make this parameter cluster-level as well? This is more convenient to implement.

Reply:
Cluster level is OK.